test/api/v3/integration/members/GET-members_id.test.js

```diff
@@ -41,6 +41,7 @@ describe('GET /members/:memberId', () => {
     expect(memberRes.stats.maxMP).to.exists;
     expect(memberRes.stats.maxHealth).to.equal(common.maxHealth);
     expect(memberRes.stats.toNextLevel).to.equal(common.tnl(memberRes.stats.lvl));
+    expect(memberRes.inbox.optOut).to.exists;
```

Comment:
I can't find anything in chai that would make it an alias to exist, so it's interesting this didn't break the test suite.